src/blocks/content-helper/related-top-posts/component-list.tsx

```diff
@@ -30,10 +29,7 @@ function RelatedTopPostList() {
 				.then( ( result ): void => {
 					const mappedPosts: RelatedTopPostData[] = result.posts.map(
 						( post: RelatedTopPostData ): RelatedTopPostData => (
-							{
-								...post,
-								date: getDateInUserLang( new Date( post.date ), SHORT_DATE_FORMAT ),
-							}
+							{ ...post }
```

Comment:
Let the final component decide how the passed date should be displayed.

Reply:
Is this mapping needed now? seems like `result` already have the required structure.